### DISCUSSION TITLE
Insert null values in Swift and JavaScript client

### DISCUSSION THREAD

**Discussion by Magnuti** [supabase/supabase#19835] [Feature Requests]
Having a table that contain nullable columns, it does not seem possible to insert rows into that table where some columns are not set (i.e., they are `NULL`). Is this a limitation of both the Swift and the JavaScript client or is there a reason for not allowing that?

**Comment by GaryAustin1** [comment]
![image](https://github.com/supabase/supabase/assets/54564956/035c735b-855f-43e2-bd3e-6aa7679cd079)
Maybe.
